File: з1.py
print("таблица значений функции y = 3x^2 + 5x/13")
print()
A=float(input("введите значение A: "))
B=float(input("введите значение B: "))
if B<A:
    print("в должно быть больше A")
    exit()
H=float(input("введите шаг H (H>0): "))
if H<=0:
    print("H должно быть больше 0")
    exit()
print()
print("|     x      |     y      |")
# считаем количество шагов
n=int((B-A)/H)+1
for i in range(n):
    x=A+i*H
    if x>B:
        x=B
    y=3*x**2+(5*x)/13
    print(f"| {x:8.4f}   | {y:8.4f}   |")

File: з1.pas
program z1;
var a,b,h,x,y:real; i,n:integer;
begin
  writeln('таблица значений функции y = 3x^2 + 5x/13');
    writeln('введите значение A: ');
    readln(A);
    writeln('введите значение B: ');
    readln(B);
if B<A then
  begin
    writeln('B должно быть больше A');
    exit;
    end;
    writeln('введите шаг H (H>0): ');
    readln(H);
    if h<=0 then
  begin
    writeln('H должно быть больше 0');
    exit;
    end;
  writeln;
  writeln('|     x     |     y     |');
  n:=trunc((B - A)/H) + 1; 
  for i:=0 to n-1 do
  begin
    x:=A+i*H;
    if x>B then
      x:=B; 
    y:=3*x*x+(5*x)/13;
    writeln('|',x:8:4, '   |',y:8:4, '   |');
  end;
end.